app/lib/services/patient_service.dart

```diff
@@ -35,4 +36,13 @@ class PatientService {
                 builder: (context) => PatientScreen(patientId: patientId)))
         .whenComplete(() => LocationService.leaveLocation());
   }
+
+  static Future<Duration> movePatient(Patient patient, Location moveTo) async {
+    const int waitTimeSeconds = 5;
+    ApiService apiService = GetIt.instance.get<ApiService>();
+    return apiService.api
+        .runActionPerformMovePatientPost(RunActionPerformMovePatientPostRequest(
+            patientId: patient.id, newLocationId: moveTo.id))
+        .then((value) => const Duration(seconds: waitTimeSeconds));
```

Comment:
According to the spec this should return a player location and patient location. I think (for the generated code) we have to specify the schema in the api.yaml, then we can use the response.  
Or isn't it implemented in the backend yet?

Reply:
changed the api spec and implemented it -> done